packages/mui-material/src/Modal/Modal.js

```diff
@@ -120,17 +117,23 @@ const Modal = React.forwardRef(function Modal(inProps, ref) {
         root: () => ({
           ...resolveComponentProps(rootSlotProps, ownerState),
           ...(!isHostComponent(RootSlot) && { as: component, theme }),
+          className: clsx(
+            className,
+            rootSlotProps?.className,
+            classes?.root,
```

Comment:
`classes` will always be defined. I guess default is empty object `{}`.
```suggestion
            classes.root,
```

Reply:
```jsx
Modal.d.ts
    /**
     * Override or extend the styles applied to the component.
     */
    classes?: Partial<ModalUnstyledClasses>;
```

The default isn't an empty object. I deem it unnecessary to add one for this change.

Reply:
There is no `@default` JSDoc tag in the type, but I saw it in [`composeClasses`](https://github.com/mui/material-ui/blob/master/packages/mui-utils/src/composeClasses/composeClasses.ts#L6) implementation. Also, I don't see it using optional chaining in other components.

Reply:
This `classses` is not the returned value of `composeClasses`. It's `classes` prop. I didn't want to add `@default JSDoc` tag to keep consistency with other components' API docs.